components/query_handler/queryHandler.py

```
@@ -1,35 +1,51 @@
 import json
 from MSG_TYPES import MSG_TYPES
+from queryFunctions import *
```

Ruff (0.8.2)

`from queryFunctions import *` used; unable to detect undefined names

(F403)